packages/block-library/src/navigation/edit/index.js

```diff
@@ -91,6 +91,10 @@ function useResponsiveMenu( navRef ) {
 		} else {
 			htmlElement.classList.remove( 'has-modal-open' );
 		}
+
+		return () => {
+			htmlElement.classList.remove( 'has-modal-open' );
```

Comment:
Will `htmlElement` always be defined. For example if the hook bales out early on the `if ( ! navRef.current ) {` check could we conceivably end up with an error.

```suggestion
			htmlElement?.classList.remove( 'has-modal-open' );
```

Reply:
Good point! I thought it wouldn't be an issue because the `useEffect` wouldn't have its cleanup if the `navRef.current` is not defined.

But thinking more, since it's about a ref I think it could be changed between the render and the cleanup.